mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/event/impl/OutboxEventLoader.java

```diff
@@ -50,7 +50,7 @@ final class OutboxEventLoader implements ToStringTreeAppendable {
 		// so we can afford to just skip events that are already locked,
 		// and process them later when they are no longer locked.
 		if ( dialect.supportsSkipLocked() ) {
-			lockOptions = new LockOptions( LockMode.PESSIMISTIC_WRITE ).setTimeOut( LockOptions.SKIP_LOCKED );
+			lockOptions = new LockOptions( LockMode.UPGRADE_SKIPLOCKED );
```

Comment:
It seems we (I?) forgot to update the comment above:

```
 		// Thus, our last option is to actually enforce locks ahead of time (LockModeType.PESSIMISTIC_WRITE),
		// and to avoid conflicts by simply never working on events that are already locked (LockOptions.SKIP_LOCKED).
```

Should be:

```
		// Thus, our last option is to actually enforce locks ahead of time (UPGRADE),
		// and to avoid conflicts by simply never working on events that are already locked (SKIP_LOCKED):
		// that's what LockMode.UPGRADE_SKIPLOCKED is for.
```